packages/styling/src/styles/getPlaceholderStyles.ts

```diff
@@ -0,0 +1,22 @@
+import { IStyle } from '@uifabric/merge-styles';
+
+/**
+ * Generates a style for placeholder for each of the browsers supported by office-ui-fabric-react.
+ * @param styles - The style to use.
+ * @returns The style object with the placeholder directive as key and the style passed in as value,
+ * for each browser depending on the placeholder directive it uses.
+ */
+export function getPlaceholderStyles(
```

Comment:
I don't think this is quite what you should do. Instead I'd recommend:
```ts
export function getPlaceholderStyles(styles: IStyle): IStyle {
  return {
    selectors: // the stuff you're currently returning from the method
  }
}
```

Then use like this:
```ts
function getStyles() {
  return {
    something: [
      { /*some styles*/ },
      getPlaceholderStyles({ /*whatever*/ })
    ]
  };
}
```

Does that make sense?

Reply:
Makes sense. I have made a commit. https://github.com/OfficeDev/office-ui-fabric-react/pull/9488/commits/86c0805d690e568cfebdeb3c0dc6ec821749c199?file-filters%5B%5D=.ts&file-filters%5B%5D=.tsx